apps/pie-docs/src/assets/styles/base/_layout.scss

```diff
@@ -65,3 +59,102 @@
         z-index: f.zIndex(low);
     }
 }
+
+// styles shared between homePage and contentPage
+.c-content {
+    flex: 1;
+    background-color: f.$color-background-default;
+    border-top: 1px solid f.$color-border-default;
+
+    & a {
+        color: f.$color-content-link-distinct;
+
+        &:visited {
+            color: f.$color-content-link-visited;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: none;
+        }
+    }
+
+    & p:has(+ .c-tokensTable) {
+        // For a tokens table directly following a description add margin.
+        // Descriptions normally have a `<hr />` as a separator
+        margin-bottom: f.spacing(f);
```

Comment:
same with these and the paddings on this page please?
`margin-block-end` and `padding-block-start/end`